IFC4x3/Properties/r/RadiationPattern_0KBK4$FZP5memrgigaPaRn/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RadiationPattern_0KBK4FZP5memrgigaPaRn" Name="RadiationPattern" UniqueId="142d413f-3e36-45c2-8c35-aacaa46646f1" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
```

Comment:
so i think properties should be specific and enums more generic so i suggest 

1. `AntennaRadiationPattern` for the property 
2. `PEnum_RadiationPattern` for the enumeration. 
consider the results of googling 2 should be suffice.